src/core.jl

```diff
@@ -226,9 +211,12 @@ function with_def_graph(ex)
     error("Improper use of with_def_graph")
     (kwargs === nothing) && (kwargs = [])
     new_args = args[2:end]
+    # Temporarily eliminate passing through keyword arguments since
+    # that breaks Revise at the moment. Should add back once Revise
+    # is patched.
```

Comment:
Can an Issue be openned to remind to fix this?
(I just stopped using Revise)

Reply:
I opened an issue on Revise and it was fixed there; I've deleted this comment on master.